### PR TITLE
Add metadata in calendar to one specific issue

### LINKED ISSUE #5106: Creation of newspaper processes: Enabling adding individual metadata for the first issue per day
### Problem
For the creation of processes for newspaper issues, to functions are available to add metadata to the processes. 
- _Add metadata to all issues_: The selected metadata field and its value are assigned to all issues. This is relevant for metadata like language, digital collection ... which are usually the same for all issues. 
- _Add metadata_: The selected metadata field and its value are assigned to only one  issues. This is relevant for metadata like issue-label ... which are not the same for all issues. 

It seems, that the function of _Add metadata to all issues_ and _Add metadata_ for the **first issue** are the same. In both cases, the metadata field is assigned to all issues and the values cannot be changed. In addition, it is not possible to delete metadata of the first issue. If the metadata is deleted, it is deleted for all all issues. 
For other issues, the function _Add metadata_ works correctly and the metadata field and its value is only assigned to one issue and the metadata field can be deleted. 

### Solution
The function  _Add metadata_ for the first issue must be corrected to enable the assignment of individual metadata for the first issue. 

### Example
ZeitungMetadatenAusgabeFehler

### PR DESCRIPTION
**Pull request by oliver-stoehr**
This pull requests adds a new button to the calendar day dialog. With this button metadata can be added to one specific issue.
For example this allows to add metadata only to the daily occurring "morning issue" without adding the metadata also to the daily occurring "evening issue".

Resolves #5106.

### CODE REVIEW

**Review by andre-hohmann — APPROVED** [review]
It is possible to create processes for several issues per day which can be labelled separately according to the issuance, as for example: 
- Morning edition
- Evening edition
- Special edition

Thus, from the user perspective, it works and the issue #5106 is resolved.

File: Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/metadata/CountableMetadata.java [hunk 1]
```diff
@@ -225,11 +251,18 @@ public String getValue(Pair<LocalDate, Issue> selectedIssue, MonthDay yearStart)
...
+        if (metadataType == null || metadataType.equals(this.metadataType)) {
+            if (this.create.getRight()) {
+                return  Objects.nonNull(issue) && Objects.equals(this.create.getMiddle(), issue.getRight());
+            } else {
+                return (null == create
+                        && new IssueComparator(block).compare(Pair.of(this.create.getLeft(), this.create.getMiddle()), issue) <= 0
                         && (delete == null || new IssueComparator(block).compare(issue, delete) < 0)
                         || Boolean.TRUE.equals(create) && this.create.equals(issue) || Boolean.FALSE.equals(create)
-                                && (issue == null && this.delete == null || issue.equals(this.delete)));
+                        && (issue == null && this.delete == null || issue.equals(this.delete)));
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/metadata/CountableMetadata.java:262]: If it is possible replace null comparing with Objects.isNull() or Objects.nonNull() method calls.

File: Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProcessesPage.java.orig [hunk 2]
```diff
@@ -0,0 +1,513 @@
+/*
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProcessesPage.java.orig:1]: Is this file needed. Looks like a local backup of a later modified file. Remove this file if its not needed anymore.

File: Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/CalendarPage.java [hunk 3]
```diff
@@ -0,0 +1,291 @@
+package org.kitodo.selenium.testframework.pages;
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/CalendarPage.java:1]: Missing licence header

**Review by henning-gerhardt — CHANGES_REQUESTED** [review]
A licence header is now missing.

**Review by henning-gerhardt — APPROVED** [review]

**Pull request merged**